demos/jsapi-integration/index.html

```diff
@@ -1,39 +1,39 @@
 <!DOCTYPE html>
 <html>
-<head>
+  <head>
   <meta charset="utf-8">
   <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no">
   <title>Share auth between rest-js and the jsapi</title>
 
-  <style>
-    html,
-    body,
-    #viewDiv {
-      padding: 0;
-      margin: 0;
-      height: 100%;
-      width: 100%;
-    }
-  </style>
+    <style>
+      html,
+      body,
+      #viewDiv {
+        padding: 0;
+        margin: 0;
+        height: 100%;
+        width: 100%;
+      }
+    </style>
 
   <link rel="stylesheet" href="https://js.arcgis.com/4.7/esri/css/main.css">
 
-  <script src="node_modules/@esri/arcgis-rest-request/dist/umd/request.umd.js"></script>
-  <script src="node_modules/@esri/arcgis-rest-auth/dist/umd/auth.umd.js"></script>
-  <script src="node_modules/@esri/arcgis-rest-portal/dist/umd/portal.umd.js"></script>
+    <script src="node_modules/@esri/arcgis-rest-request/dist/umd/request.umd.js"></script>
+    <script src="node_modules/@esri/arcgis-rest-auth/dist/umd/auth.umd.js"></script>
+    <script src="node_modules/@esri/arcgis-rest-portal/dist/umd/portal.umd.js"></script>
 
-  <!--
+    <!--
     NOTE: rather than include the ArcGIS API for JavaScript via a <script> tag
     typically you would using something like https://github.com/Esri/esri-loader
     to lazy-load the API _after_ you had used the light-weight arcgis-rest-js
     libraries above to log in and fetch some data (see NOTE below)
   -->
-  <script src="https://js.arcgis.com/4.7/"></script>
-</head>
+    <script src="https://js.arcgis.com/4.7/"></script>
```

Comment:
might as well bump to the latest here
```suggestion
    <script src="https://js.arcgis.com/4.13/"></script>
```

Reply:
done. bumped to 4.13 for jsapi and css.